src/neo/Wallets/Wallet.cs

```diff
@@ -241,7 +241,7 @@ public virtual WalletAccount Import(string nep2, string passphrase, int N = 1638
             return account;
         }
 
-        public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null, Signer[] cosigners = null)
+        public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null, Signer[] cosigners = null, DataCache snapshot = null)
```

Comment:
Why do we need this?

Reply:
It's for UT, in order to fake the balance without commit the snapshot

Reply:
We shouldn't change the APIs for UT only.

Reply:
Changed